core/iwasm/aot/aot_runtime.c

```diff
@@ -1444,8 +1458,19 @@ aot_call_function(WASMExecEnv *exec_env, AOTFunctionInstance *function,
         }
 #endif
 
-        ret = invoke_native_internal(exec_env, function->u.func.func_ptr,
-                                     func_type, NULL, NULL, argv, argc, argv);
+#ifdef OS_ENABLE_BLOCK_INSN_INTERRUPT
+        wasm_exec_env_push_jmpbuf(exec_env, &jmpbuf_node);
+        wasm_runtime_set_exec_env_tls(exec_env);
+        if (os_setjmp(jmpbuf_node.jmpbuf) == 0)
+#endif
+            ret =
+                invoke_native_internal(exec_env, function->u.func.func_ptr,
+                                       func_type, NULL, NULL, argv, argc, argv);
```

Comment:
How about add a pair of braces? Otherwise it may be easy to make a mistake when adding new lines

```suggestion
        {
            ret =
                invoke_native_internal(exec_env, function->u.func.func_ptr,
                                       func_type, NULL, NULL, argv, argc, argv);
        }
```

Reply:
not a big deal since the code formatter would show that with the indentation; I'm adding it anyway since it makes the code cleaner with the `else` statement